DigitalLearningSolutions.Web/ViewModels/Login/LoginViewModel.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace DigitalLearningSolutions.Web.ViewModels.Login
+{
+    using System.ComponentModel.DataAnnotations;
+
+    public class LoginViewModel
+    {
+        public LoginViewModel()
+        {
+            Username = string.Empty;
+            Password = string.Empty;
+            RememberMe = false;
+        }
+
+        [Required]
```

Comment:
Could we potentially modify the error messages that the user gets to be something like "Please enter a username to log in" rather than using "XXX field"?